packages/scipy/meta.yaml

```diff
@@ -1,60 +1,92 @@
 package:
   name: scipy
-  version: 0.17.1
+  version: 1.7.3
```

Comment:
It would also be nice to get rid of the second install. There is an issue that downstream packages that use cython don't see our modified cython headers. This is why I currently have to patch scikit-learn to cythonize the files with an include path pointing to our modified scipy. This worked fine before but now it just doesn't seem to install our modified scipy the right way. We could always set `skip_host=False` but that seems pretty expensive.